lib/main.dart

```diff
@@ -39,10 +40,11 @@ class MyApp extends StatelessWidget {
         colorScheme: ColorScheme.fromSeed(seedColor: Colors.deepPurple),
         useMaterial3: true,
       ),
-      home: SignupScreen(),
+      home: SignInScreen(),
       routes: {
         "/sign-up": (context) => SignupScreen(),
-        "/sign-in": (context) => MyHomePage(title: "title")
+        "/sign-in": (context) => SignInScreen(),
+        "/overview": (context) => MyHomePage(title: "title")
```

Comment:
I'm kinda allergic to random strings -> we should change that later at least to constants or even to a map (route <-> screen)